### ISSUE TITLE
Nyquist plugins sometimes don't follow Audacity's language, use system locale (?) instead

### ISSUE THREAD

**Issue by LWinterberg** [audacity/audacity#2764]
This has been reproduced by @dozzzzer and me on the following systems: 

* System 1: Windows 11, Audacity set to English, OS language German, OS locale German - Nyquist listings in German
* System 2: ??, Audacity set to English, OS language Russian, OS locale Polish - Nyquist listings in Polish. 

<details>
<summary>Example screenshot:</summary>
 
![image](https://user-images.githubusercontent.com/87814144/161508109-6b6d7069-f95e-4e1f-9ada-2104b80d3961.png)

There are 3 versions per effect due to #1381 (3.1.3, master, a PR build), 2 of them are English, one is German. 
</details>

**Comment by Paul-Licameli** [comment]
What is “Nyquist Listings”?  What appears in menus?

There is localization of message boxes produced by Nyquist effects, but I would expect that to follow the language chosen in Interface preferences.

**Comment by LWinterberg** [comment]
yes, their name in the menus. the effects themselves are in the audacity language

It might be easier to see in a shorter menu than the main effects window: 
![image](https://user-images.githubusercontent.com/87814144/161536655-30ea1568-49c5-4fe4-8ebb-20cc0203b97d.png)

**Comment by teszNU** [comment]
Well, I found lackd translation bugs this, but my os languages is chinese. the newest version normalizations.
![无标题](https://user-images.githubusercontent.com/61547678/197398179-798a8c9e-0c58-4fcf-aa0a-85784e09d167.png)

and, PLZ resolved the website problem by i form china:-( 
without VPN visit <***.audacityteam.org>, error 1030, known cloudflame AI banned china ip area to access.
with VPN visit <***.audacityteam.org>, normal access.
cloudflame track code here: 75eb1d3ae86fdbaa

**Comment by TuulikkiUnelma** [comment]
This happens to me as well on Linux. I've set it so that all language should be in English, while things like the keyboard, dates, and other things are Finnish, yet Audacity chose Finnish as it's default language and some plugins keep being Finnish
